File: CPP/prawie_jak_fibonacci_prawie.cpp
#include <iostream>
#include <vector>
using namespace std;



int main()
{
    int n, temp, ile = 0;
    vector<float> wejscie;

    cin >> n;
    for (int i = 0; i < n; i++)
    {
        cin >> temp;
        wejscie.push_back(temp);
    }

    for(int i = 0; i < n; i++)
    {
        if (( wejscie[i+3] ) == (( wejscie[i+2] ) + ( wejscie[i+1] )))
        {
            ile += 1;
        }

    }
    cout << ile;
    return 0;
}

File: PY/prawie_jak_fibonacci_prawie.py
n = int(input())
ile = 0

wejscie = input().split()
for i in range(n-3):
    if int(wejscie[i+3]) == (int(wejscie[i+2]) + int(wejscie[i+1])):
        ile += 1

print(ile)
quit()